pkg/nf90io/nf90io_utils.F

```diff
@@ -0,0 +1,376 @@
+#include "PACKAGES_CONFIG.h"
+#include "CPP_OPTIONS.h"
+
+CCCCCCCCCCCCCCC
+      FUNCTION NF90IO_VAR_PAR_ACCESS(ncid, varid)
+
+      use netcdf
+#include "SIZE.h"
+#include "EEPARAMS.h"
+#include "EESUPPORT.h"
+
```

Comment:
I had to `#include "NF90IO.h"` here and in `NF90IO_OPEN` for this to compile.  I think generally we declare functions and their type explicitly in every subroutine that uses them (i.e., a `FUNCTION` and a type statement), so there may not be a need for this header.  But maybe we should discuss this convention @jm-c, @christophernhill?

Reply:
Oh, hmmm.  Maybe you use a more strict compile options than me?

Reply:
Hmm, I am using stock linux_amd64_gfortran...